### PR TITLE
tier up +dmg +speed -minus speed

### PR DESCRIPTION
**Pull request by Zhaylo**
I brought the steppe light mace up in le1ngth and took the trust off and brough it's tier up. Rebalanced the cata mace to be long and medium dmg and speed. and the militia pernach to be a high tier short and fast mace.

### CODE REVIEW

File: items.json [hunk 1]
```diff
@@ -9400,18 +9400,18 @@
...
+        "handling": 87,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon"
         ],
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
-        "thrustSpeed": 87,
+        "thrustSpeed": 88,
         "swingDamage": 10,
```
Comment (namidaka) [inline items.json:9412]: is this low damage and tier on purpose?

File: items.json [hunk 2]
```diff
@@ -10400,18 +10400,18 @@
...
         ],
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
-        "thrustSpeed": 90,
-        "swingDamage": 15,
+        "thrustSpeed": 92,
+        "swingDamage": 23,
         "swingDamageType": "Blunt",
-        "swingSpeed": 85
+        "swingSpeed": 88
```
Comment (namidaka) [inline items.json:10414]: can you go for lower swing speed higher damage?

File: items.json [hunk 3]
```diff
@@ -15252,9 +15252,9 @@
     "name": "Heavy Flanged Mace",
     "culture": "Khuzait",
     "type": "OneHandedWeapon",
-    "price": 198,
-    "weight": 2.29,
-    "tier": 0.7856775,
+    "price": 2020,
+    "weight": 1.97,
+    "tier": 4.66271448,
```
Comment (namidaka) [inline items.json:15257]: is this supposed to be a mace for beginners? If the answer is yes i'm fine with it, just want to double check

**Review by namidaka — CHANGES_REQUESTED** [review]

File: items.json [hunk 4]
```diff
@@ -10400,18 +10400,18 @@
...
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon"
         ],
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
-        "thrustSpeed": 90,
-        "swingDamage": 15,
+        "thrustSpeed": 92,
+        "swingDamage": 23,
```
Comment (namidaka) [inline items.json:10412]: can you go for lower swing speed higher damage?

**Review by namidaka — CHANGES_REQUESTED** [review]

File: items.json [hunk 5]
```diff
@@ -9387,9 +9387,9 @@
...
     "type": "OneHandedWeapon",
-    "price": 277,
-    "weight": 1.88,
-    "tier": 1.09812713,
+    "price": 298,
+    "weight": 1.78,
+    "tier": 1.17355168,
     "requirement": 0,
     "flags": [],
     "weapons": [
```
Comment (namidaka) [inline items.json:9395]: You forgot to change the damage

File: items.json (hunk 2 above)
Comment (namidaka) [inline items.json:10414]: can you go for lower swing speed higher damage?

File: items.json [hunk 6]
```diff
@@ -15252,9 +15252,9 @@
...
     "type": "OneHandedWeapon",
-    "price": 198,
-    "weight": 2.29,
-    "tier": 0.7856775,
+    "price": 2020,
+    "weight": 1.97,
+    "tier": 4.66271448,
     "requirement": 0,
     "flags": [],
     "weapons": [
```
Comment (namidaka) [inline items.json:15260]: is this begginner tier on purpose? If the answer is yes i'm fine with it

**Review by namidaka — CHANGES_REQUESTED** [review]

File: items.json (hunk 6 above)
Comment (Zhaylo) [reply items.json:15260]: Made the tier a little higher

File: items.json (hunk 2 above)
Comment (Zhaylo) [reply items.json:10414]: made it slower and stronger

File: items.json (hunk 5 above)
Comment (Zhaylo) [reply items.json:9395]: changed the damage to 21

**Pull request closed without merging**